src/main/java/ui/MenuControl.java

```diff
@@ -89,47 +86,69 @@ private Menu createFileMenu() {
     }
 
     /**
-     * Logs the user action and creates a new board
+     * Checks whether the current open board has unsaved changes
+     * @return {@code true} if current open board is dirty, {@code false} otherwise.
```

Comment:
Don't use `{@code}` for `true` and `false`. Also, should define in comment about what is dirty, e.g. `... board is dirty, i.e. has unsaved changes`.